src/pages/Home/Home.jsx

```diff
@@ -4,10 +4,14 @@ import "./Home.css";
 import food from "../../assets/photo-1475090169767-40ed8d18f67d-removebg-preview.png";
 import { Link } from "react-router-dom";
 import axios from "axios";
-import { useStateContext } from "../../hooks";
 import { ACTION_TYPES } from "../../reducer";
 import { CategoryCard } from "../../components";
-import { useAuthContext, useAuth } from "../../hooks";
+import {
+  useAuthContext,

```

Comment:
Great use of barrel imports.

Reply:
Thanks for suggestion, will implement those.